src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager_impl.h

```diff
@@ -56,6 +56,14 @@ class ResourceAllocationManagerImpl : public ResourceAllocationManager {
 
   void ResetAllAllocations() FINAL;
 
+  /**
+   * @brief GetApplicationExtention Provides access to application RC extention
+   * @param application Application
+   * @return Pointer to RC extention of application or NULL if not available
+   */
+  RCAppExtensionPtr GetApplicationExtention(
```

Comment:
@LitvinenkoIra please move this funciton to RCCOmmandRequest

Reply:
@LuxoftAKutsan not now, sorry